v3/examples/sqlite/spin.toml

```diff
@@ -0,0 +1,19 @@
+spin_manifest_version = 2
+
+[application]
+name = "sqlite-example"
+version = "0.1.0"
+authors = ["Fermyon Engineering <engineering@fermyon.com>"]
+description = ""
+
+[[trigger.http]]
+route = "/..."
+component = "sqlite"
+
+[component.sqlite]
+source = "main.wasm"
+allowed_outbound_hosts = []
+sqlite_databases = ["default"]
+[component.sqlite.build]
+command = "tinygo build -target=wasip2 --wit-package $(go list -mod=readonly -m -f '{{.Dir}}' github.com/spinframework/spin-go-sdk/v3)/wit --wit-world http-trigger -gc=leaking -o main.wasm main.go"
```

Comment:
Will the `$(go list ...)` work on Windows?

Reply:
hmm, `go list` should work, but not sure about `$(go list...)`. I had opened a ticket https://github.com/ydnar/wasi-http-go/issues/22 to check with other folks if there is a better way to do this.